hhnk_threedi_tools/core/schematisation_builder/DAMO_exporter.py

```diff
@@ -0,0 +1,52 @@
+# %%
+"""DAMO exporter based on model extent"""
+
+import geopandas as gpd
+from hhnk_research_tools.sql_functions import (
+    database_to_gdf,
+    sql_builder_select_by_location,
+)
+from hhnk_threedi_plugin.local_settings import DATABASES
+from shapely.geometry import box
+
+
+def DAMO_exporter(model_extent, table_names, EPSG_CODE="28992"):
+    # if instead of a model_extent, polder_name is given as input.
+    # make it possible to select geometry of polder from polderclusters.gpkg
+    db_dicts = {
+        "aquaprd": DATABASES.get("aquaprd_lezen", None),
+        "bgt": DATABASES.get("bgt_lezen", None),
+    }
+
+    schema = "DAMO_W"
+
+    # make bbox --> to simply string request to DAMO db
+    bbox_model = box(*model_extent.total_bounds)
```

Comment:
Het lijkt me netter om de bounding box functionaliteit toe te voegen aan research tools. Dan zorgen we ervoor dat die functie altijd werkt.